docs/pages/getting-started.md

```diff
@@ -5,10 +5,11 @@ sidebar: left-navigation-sidebar
 toc: false
 permalink: getting-started.html
 folder:
-summary: SAP Fiori Fundamentals is a light-weight presentation layer that can be used with your UI framework of choice (e.g. Angular, React, Vue, etc.). With Fiori Fundamentals’ library of stylesheets and HTML tags, developers can build consistent Fiori apps in any web-based technology.
+summary:
 
 ---
-
+ SAP Fiori Fundamentals is a light-weight presentation layer using HTML and CSS with the following implementations under active development: [Angular](https://sap.github.io/fundamental-ngx/), [React](https://sap.github.io/fundamental-react/), and [Vue](https://sap.github.io/fundamental-vue/). With Fiori Fundamentals’ library of stylesheets and HTML tags, developers can build consistent Fiori apps in any web-based technology.
+ 
```

Comment:
I'm not sure if we're using Kramdown or what here, but we should look into how to make these open in a new tab. In Kramdown, we can do `[word](www.link.com){:target="_blank"}`

Reply:
Can't do that in Markdown.

Reply:
So this is using rouge to parse the markdown, kramdown is another option we can use. See the example in my above comment.

Reply:
Okay, I see that the gem for Kramdown has already been installed so I will use that syntax.